cdap-app-fabric/src/main/java/org/apache/hadoop/mapred/LocalJobRunnerWithFix.java

```diff
@@ -559,7 +560,8 @@ public void run() {
           LOG.warn("Job {} was killed", id, t);
         } else {
           this.status.setRunState(JobStatus.FAILED);
-          LOG.error("Job {} failed", id, t);
+          LOG.error("Job Id '{}' failed. Error: ", id.getId(), ExceptionUtils.getRootCause(t));
+          LOG.debug("Job Id '{}' failed. Complete Error: ", id.getId(), t);
```

Comment:
@chtyim the idea was to only log the root cause stack trace as error, but still have the complete available in the debugs. Please let me know your thoughts.